kosat-core/src/commonMain/kotlin/org/kosat/Clause.kt

```diff
@@ -27,4 +27,13 @@ data class Clause(
             return Clause(lits)
         }
     }
+
+    override fun toString(): String {
+        val flags = sequenceOf(
+            if (learnt) "L" else ".",
+            if (deleted) "D" else "."
+        ).joinToString("")
+
+        return "Cl(${lits.joinToString(" ")} | flg=$flags act=${activity.round(2)} lbd=$lbd)"
```

Comment:
I do not like this format. If you ever need this in some reporting, make a custom extension function `fun Clause.toMyAwesomeFormat(): String`. Leave `toString` for standard data-class-like formatting.